### PR TITLE
Add utilities for working with time information

### PR DESCRIPTION
**Pull request by okucheruk**
This PR adds utilities for working with time information.

The `Time` class implements `currentTimeZone()` and `currentTime()` methods intended to be used in the file processes and authentication-related operations.

All possible usage occurrences in the project have been refactored.

### CODE REVIEW

File: server/src/test/java/io/javaclasses/filehub/user/AuthenticatedUserProcessTest.java [hunk 1]
```diff
@@ -51,8 +52,7 @@ void testCommandWithNullToken() {
     @Test
     @DisplayName("throw an exception if passed command has an expired authentication token")
     void testCommandWithExpiredToken() {
-        LocalDateTime expiredTime = LocalDateTime.now(ZoneId.of("UTC"))
-                                                 .minusDays(1);
+        LocalDateTime expiredTime = currentTime().minusDays(1);
```
Comment (Oleh-Stefanovych) [inline server/src/test/java/io/javaclasses/filehub/user/AuthenticatedUserProcessTest.java]: Let's remove unnecessary import since we don't use `ZoneId`.

Comment (okucheruk) [reply server/src/test/java/io/javaclasses/filehub/user/AuthenticatedUserProcessTest.java]: Pushed dbfd98e.

File: server/src/main/java/io/javaclasses/filehub/Time.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    /**
+     * Obtains the current time in UTC.
+     */
+    public static LocalDateTime currentTime() {
+        return LocalDateTime.now(currentTimeZone());
+    }
+
+    /**
+     * Obtains the current time zone ID.
```
Comment (denysLavrienov) [inline server/src/main/java/io/javaclasses/filehub/Time.java]: Please rewrite, `UTC` is not the current time zone, you can write `Obtains the {@code UTC} time zone ID.`

File: server/src/main/java/io/javaclasses/filehub/Time.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ * Utilities for working with time information.
+ */
+public final class Time {
+
+    /** Prevents instantiation of this utility class. */
+    private Time() {
+    }
+
+    /**
+     * Obtains the current time in UTC.
```
Comment (denysLavrienov) [inline server/src/main/java/io/javaclasses/filehub/Time.java]: You can write `...current time using the {@code UTC} time zone`.

File: server/src/main/java/io/javaclasses/filehub/Time.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
+package io.javaclasses.filehub;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+/**
+ * Utilities for working with time information.
```
Comment (denysLavrienov) [inline server/src/main/java/io/javaclasses/filehub/Time.java]: Please rewrite:

image

File: server/src/main/java/io/javaclasses/filehub/Time.java (hunk 2 above)
Comment (okucheruk) [reply server/src/main/java/io/javaclasses/filehub/Time.java]: Javadoc has been updated.

File: server/src/main/java/io/javaclasses/filehub/Time.java (hunk 4 above)
Comment (okucheruk) [reply server/src/main/java/io/javaclasses/filehub/Time.java]: There is nothing wrong with this Javadoc.

File: server/src/main/java/io/javaclasses/filehub/Time.java (hunk 3 above)
Comment (okucheruk) [reply server/src/main/java/io/javaclasses/filehub/Time.java]: It's not the UTC time zone but an abbreviation of "Coordinated Universal Time".

**Pull request merged**